### PR TITLE
Like music

### PR DESCRIPTION
**Pull request by manh2911**
https://edu-redmine.sun-asterisk.vn/issues/12294

### CODE REVIEW

**Comment by manh2911** [comment]
ready

File: resources/js/favorite.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+        return false;
+    });
+
+    $('body').on('click', unfavoriteID, function() {
+        $.ajax({
+            type: 'POST',
+            url: '/favorite/unlike',
+            data: "songId=" + songId + "&user_ID=" + user_ID,
+            success: function() {
+                $templateLike = `<i class="fa fa-heart-o text favorite" id="favorite-`+ songId +`"></i>`
```
Comment (BunnyPi04) [inline resources/js/favorite.js]: convention

**Comment by manh2911** [comment]
ready

File: app/Http/Controllers/FavoriteController.php [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+        $favorite->user_id = $request->user_ID;
+        $favorite->music_id = $request->songId;
+        $favorite->save();
+    }
+
+    public function unlike(Request $request){
+        $user_id = $request->user_ID;
+        $music_id = $request->songId;
+        Favorite::where('user_id',$user_id)
+            ->where('music_id',$music_id)
```
Comment (BunnyPi04) [inline app/Http/Controllers/FavoriteController.php]: convention

File: app/Http/Controllers/FavoriteController.php [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+use App\Favorite;
+use Illuminate\Http\Request;
+
+class FavoriteController extends Controller
+{
+    public function like(Request $request){
+        $favorite = new Favorite;
+        $favorite->user_id = $request->user_ID;
+        $favorite->music_id = $request->songId;
```
Comment (BunnyPi04) [inline app/Http/Controllers/FavoriteController.php]: song_id nhé

File: app/Http/Controllers/FavoriteController.php [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+namespace App\Http\Controllers;
+
+use App\Favorite;
+use Illuminate\Http\Request;
+
+class FavoriteController extends Controller
+{
+    public function like(Request $request){
+        $favorite = new Favorite;
+        $favorite->user_id = $request->user_ID;
```
Comment (BunnyPi04) [inline app/Http/Controllers/FavoriteController.php]: $request->user_id nhé

File: app/Http/Controllers/FavoriteController.php [hunk 5]
```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Favorite;
+use Illuminate\Http\Request;
+
+class FavoriteController extends Controller
+{
+    public function like(Request $request){
```
Comment (BunnyPi04) [inline app/Http/Controllers/FavoriteController.php]: convention

**Review by BunnyPi04 — COMMENTED** [review]
check các chỗ tương tự

**Pull request merged**